lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -266,6 +266,9 @@ protected function transformCampaign($data) {
         $output['uri'] = $data->uri;
       }
 
+      if (dosomething_helpers_get_variable('node', $data->id, 'magic_link_copy')) {
```

Comment:
@angaither In the new commit I updated it to grab the variable first, and then it's either `FALSE` or the value. Does that work?